frontend/packages/ux-editor/src/components/config/editModal/EditNumberValue.tsx

```diff
@@ -2,9 +2,9 @@ import React from 'react';
 import type { IGenericEditComponent } from '../componentConfig';
 import { useTranslation } from 'react-i18next';
 import { FormField } from '../../FormField';
-import { Textfield } from '@digdir/design-system-react';
 import { getComponentPropertyLabel } from '../../../utils/language';
 import { setComponentProperty } from '../../../utils/component';
+import { StudioDecimalInput } from '../../../../../../libs/studio-components/src/components/StudioNumberInput/StudioDecimalInput';
```

Comment:
Veldig fint at du har flyttet komponenten inn i den nye pakken. Her bør du kunne importere den på denne måten:
```suggestion
import { StudioDecimalInput } from '@altinn/studio-components';
```
Imidlertid må du også eksportere den i [indeksfilen i `components`](https://github.com/Altinn/altinn-studio/blob/master/frontend/libs/studio-components/src/components/index.ts) for at dette skal fungere. Se hvordan det er gjort med `StudioModal`.